examples/siro_sandbox/README_VR.md

```diff
@@ -0,0 +1,218 @@
+# VR Human-in-the-loop (HITL) Evaluation
+
+**Note: VR HITL evaluation currently only works from this branch.**
+
+HITL evaluation can be driven from VR. In this mode, the sandbox app acts as a server that can be remotely accessed by a client.
+
+As it stands, the VR integration can only be used with the `fetch` app state. In this mode, the user controls a human avatar that can play fetch with a policy-driven Spot robot.
+
+## Table of Contents
+- [VR Human-in-the-loop (HITL) Evaluation](#vr-human-in-the-loop-hitl-evaluation)
+  - [Table of Contents](#table-of-contents)
+  - [Installation](#installation)
+    - [Requirements](#requirements)
+      - [Server](#server)
+      - [Client](#client)
+  - [Launch Commands](#launch-commands)
+      - [Example](#example)
+  - [Unity Data Folder](#unity-data-folder)
+    - [Dataset Processing](#dataset-processing)
+      - [Installation](#installation-1)
+      - [Usage](#usage)
+      - [Importing Data Into Unity](#importing-data-into-unity)
+  - [Connection](#connection)
+  - [Server Controls](#server-controls)
+  - [Troubleshooting](#troubleshooting)
+    - [Deployment to the VR Headset](#deployment-to-the-vr-headset)
+    - [Connection Issues](#connection-issues)
+    - [Slow Performance](#slow-performance)
+  - [Other information](#other-information)
+    - [Serving HTTPS content from your local machine](#serving-https-content-from-your-local-machine)
+
+
+## Installation
+
+The system is composed of the following components:
+
+* The **Server**, which is the sandbox app.
+* The **Client**, which runs on Unity and is deployed on a VR headset.
+
+### Requirements
+
+#### Server
+
+| Requirements | Notes |
+|---|---|
+| `habitat-sim` | Use the `main` branch. Bullet is required. |
+| `habitat-lab` | This specific `habitat-lab` version must be installed. See [instructions](../../README.md#installation). The `main` branch is currently incompatible. |
+| Datasets | `python -m habitat_sim.utils.datasets_download --uids habitat_humanoids hab_spot_arm ycb hssd-hab --data-path data/` |
+| [hssd-models](https://huggingface.co/datasets/hssd/hssd-models) | Required for dataset processing. |
+| [habitat_humanoids](https://huggingface.co/datasets/ai-habitat/habitat_humanoids) | Use the `main` branch. |
+| [Server files](https://huggingface.co/datasets/ai-habitat/siro_fetch_extra_data) | Copy the files as-is to `data/`, following the directory structure. |
+| `websockets` | `pip install websockets` |
+
+#### Client
+
+| Requirements | Notes |
+|---|---|
+| Unity 2022.3.7f1 | Beware that a license may be required by your organization. |
+| Quest Headset | Tested on Quest Pro and Quest 3. Make sure that [developer mode](https://developer.oculus.com/documentation/native/android/mobile-device-setup/) is activated. |
+| [siro_hitl_unity_client](https://github.com/eundersander/siro_hitl_unity_client) | Use the `main` branch. See [instructions](https://github.com/eundersander/siro_hitl_unity_client/blob/main/README.md). |
```

Comment:
It's a little confusing how the VR readme instructions are scattered between this file and https://github.com/eundersander/siro_hitl_unity_client/blob/main/README.md. Can we consolidate?